cogs5e/initTracker.py

```diff
@@ -800,16 +796,26 @@ def mod_or_set(opt_name, old_value):
         duration = args.last('dur', -1, int)
         conc = args.last('conc', False, bool)
         end = args.last('end', False, bool)
+        parent = args.last('parent')
+
+        if parent is not None:
+            parent = parent.split('|', 1)
+            if not len(parent) == 2:
+                raise InvalidArgument("`parent` arg must be formatted `COMBATANT|EFFECT_NAME`")
+            p_combatant = await combat.select_combatant(parent[0], choice_message="Select the combatant with the parented effect.")
+            parent = await p_combatant.select_effect(parent[1])
 
         embed = EmbedWithAuthor(ctx)
         for combatant in targets:
             if effect_name.lower() in (e.name.lower() for e in combatant.get_effects()):
                 out = "Effect already exists."
             else:
-                effectObj = Effect.new(combat, combatant, duration=duration, name=effect_name, effect_args=args,
-                                       concentration=conc, tick_on_end=end)
-                result = combatant.add_effect(effectObj)
-                out = "Added effect {} to {}.".format(effect_name, combatant.name)
+                effect_obj = Effect.new(combat, combatant, duration=duration, name=effect_name, effect_args=args,
+                                        concentration=conc, tick_on_end=end)
+                result = combatant.add_effect(effect_obj)
+                if parent:
+                    combatant.get_effect(effect_name).set_parent(parent)
```

Comment:
```suggestion
                    effect_obj.set_parent(parent)
```
You already have a pointer to the existing Effect object - you don't need to get another.